core/src/main/assets/js/turbo.js

```diff
@@ -206,8 +206,10 @@
     while (element) {
       const canScroll = element.scrollHeight > element.clientHeight
       const overflowY = window.getComputedStyle(element).overflowY
+      const isScrollable = canScroll && (overflowY === "scroll" || overflowY === "auto")
+      const preventPullToRefresh = !!element.closest("[data-hotwire-native-prevent-pull-to-refresh]")
```

Comment:
`data-hotwire-native-prevent-pull-to-refresh` is quite a mouthful, I'm open to suggestions for a better name. 
I chose this name to be as descriptive as possible, so it's less confusing when reading the web app code.

Reply:
Thanks so much for this solution @leonvogt. Even though iOS doesn't need this attribute, an available solution on Android is appropriate. 

Like you said, the current attribute name is long, so I'd suggest just dropping the `-hotwire` part. So, it'd be `data-native-prevent-pull-to-refresh` instead. This is similar to bridge markup, where attributes use the `data-bridge-*` prefix. We can establish here that the `data-native-*` prefix can be used for non-bridge related attributes for the Hotwire Native libraries.

If you make that change, I'll happily merge this over 👍

Reply:
Hi @jayohms, thanks for your feedback! 
I like the attribute name `data-native-prevent-pull-to-refresh`.
I think `data-native-*` is a good prefix for the native libraries. Should still be clear enough, when reading the web code.
I'll push the changes shortly.